cli/dcoscli/data/help/marathon.txt

```diff
@@ -28,6 +28,7 @@ Usage:
     dcos marathon group show [--group-version=<group-version>] <group-id>
     dcos marathon group remove [--force] <group-id>
     dcos marathon group update [--force] <group-id> [<properties>...]
+    dcos marathon plugin [--json]
```

Comment:
here I meant `dcos marathon plugin [--json]`

Reply:
yeah... that makes sense and is symmetrical